config.dsc

```diff
@@ -171,12 +171,12 @@ config({
                 // xUnit
                 { id: "xunit.abstractions", version: "2.0.3", tfm: ".NETStandard2.0" },
                 { id: "xunit.analyzers", version: "0.10.0" },
-                { id: "xunit.assert", version: "2.4.1" },
-                { id: "xunit.core", version: "2.4.1" },
-                { id: "xunit.extensibility.core", version: "2.4.1" },
-                { id: "xunit.extensibility.execution", version: "2.4.1" },
-                { id: "xunit.runner.utility", version: "2.4.1" },
-                { id: "xunit.runner.console", version: "2.4.1" },
+                { id: "xunit.assert", version: "2.4.1-ms" },

```

Comment:
Maybe leave a comment here that this is our own fork of XUnit (and maybe leave the git url of the fork) as well as the nuget feed where we manually published this